dev/com.ibm.websphere.appserver.features/visibility/public/jsf-3.0/io.openliberty.faces-3.0.feature

```diff
@@ -0,0 +1,59 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
Since the name of this feature is faces-3.0, it should also be in a faces-3.0 directory not jsf-3.0

Reply:
renamedd